docs/installingNative.md

```diff
@@ -2,13 +2,15 @@
 
 This page describes the procedure to install and run PrairieLearn fully natively without using Docker. Certain features, such as external graders and workspaces, still require Docker. PrairieLearn supports native execution on macOS, Linux, and Windows inside [WSL 2](https://learn.microsoft.com/en-us/windows/wsl/install).
 
+## Installation
+
 - Install the prerequisites:
 
   - [Git](https://git-scm.com)
   - [Node.js 20](https://nodejs.org)
   - [Yarn](https://yarnpkg.com)
   - [Python 3.10](https://www.python.org)
-  - [PostgreSQL 16](https://www.postgresql.org)
+  - [PostgreSQL 17](https://www.postgresql.org)
```

Comment:
As I mentioned elsewhere, this is incorrect.

Reply:
Fixed in 11571eaf4cb629f088c608b49baefe0e835027bd.